### ISSUE TITLE
Cannot batch taxonomy store calls

### ISSUE THREAD

**Issue by erkanch** [pnp/pnpjs#3120]
### Major Version

3.x

### Minor Version Number

3.23

### Target environment

SharePoint Framework

### Additional environment details

SPFx 1.15

### Expected or Desired Behavior

Trying to get all terms for 3 term sets using batching

### Observed Behavior

I get Invalid request message when I use a batched SPFI object. termStore endpoints are all _api/v2.1/ endpoints, whereas the $batch endpoint is 1.0 (_api/$batch). I tried adding a custom behavior and updating the $batch url to _api/v2.1/$batch but it causes a 403 (OAuth only flow is enabled and the call has not been issued from an app.)

{"error":{"code":"-1, Microsoft.SharePoint.Client.InvalidClientQueryException","message":"Invalid request."}}
--batchresponse_92f59cfb-d018-424a-bbde-80383c64590a
Content-Type: application/http
Content-Transfer-Encoding: binary

### Steps to Reproduce

const [batchedSP, execute] = this.sp.batched();
      let translations = {};

      for (const termSet of termSetNameOrIDs) {
        // list all the terms that are direct children of this set
        batchedSP.termStore.groups
          .getById(groupId).sets
          .getById(termSet).children()
          .then(res => translations = {...translations, ...res});
      }

      await execute();

**Comment by patrick-rodgers** [comment]
Hi @erkanch - for v3 we were calling the internal endpoint as when we first did the work taxonomy was not yet on Graph. Now that it is [supported in Graph](https://learn.microsoft.com/graph/api/termstore-store-get) we have moved that functionality to the pnp js graph lib and encourage you to upgrade to v4.

The reason batching doesn't work is that taxonomy goes against _api/v2.1 and batching hits _api/$batch.
